src/backend/src/controllers/change-requests.controllers.ts

```diff
@@ -51,7 +51,7 @@ export const reviewChangeRequest = async (req: Request, res: Response) => {
   if (foundCR.accepted) return res.status(400).json({ message: `This change request is already approved!` });
 
   // verify that the user is not reviewing their own change request
-  if (reviewerId === foundCR.submitterId) return res.status(401).json({ message: 'Access Denied' });
```

Comment:
I also feel like it'd make more sense to turn this error message into a constant somewhere or create an error class we could return, so if we ever do want to change error messages, we can do so in one place.

Reply:
yeah true i should make some tickets for that